schemas/questions/types/calculated.json

```diff
@@ -43,8 +43,15 @@
               "description": "The id of an answer from which to obtain the total to validate against"
             },
             "value": {
-              "type": "integer",
-              "description": "A hard coded total to validate against"
+              "description": "A hard coded total to validate against or a value source",
+              "oneOf": [
+                {
+                  "type": "integer"
```

Comment:
Any reason to not allow `number` which allows floating points. If it works fine in runner, we should allow either.

Reply:
Added `number`, thanks!